### PR TITLE
Portfolioform website

### PR DESCRIPTION
**Pull request by stevenmukama**
Added form to the portfolio website.
Added  CSS styles to the form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+                            <ul><button type="button" class="button7"> bootstrap</button></ul>
+                            <ul><button type="button" class="button8"> Ruby</button></ul>
+                        </li>
+                    </div>
+                </div>
+                <button type="button" class="button9"> See Project</button>
+            </div>
+            <div class="card">
+                <!-- second card -->
+                <div class="listproject">
```
Comment (christianbiring1) [inline index.html:79]: - [ ] Kindly make sure the page width at all levels fits the screen size.
Your design :point_down: 
Screenshot from 2022-11-01 18-48-12

File: index.html [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                        required. has been
+                        the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                        standard dummy text.</p>
+
+                    <li class="listbutton">
+                        <ul><button type="button" class="button1"> css</button></ul>
+                        <ul><button type="button" class="button2"> html</button></ul>
+                        <ul><button type="button" class="button3"> bootstrap</button></ul>
+                        <ul><button type="button" class="button4"> Ruby</button></ul>
+                    </li>
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_button.asp) `when to use a button`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+                <div class="card">
+                    <div class="listproject">
+                        <h2 class="art">Profesional Art Printing Data</h2>
+                        <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                            required. has been the
+                            industry's standard</p>
+                        <li class="listbutton2">
+                            <ul><button type="button" class="button6"> html</button></ul>
+                            <ul><button type="button" class="button7"> bootstrap</button></ul>
+                            <ul><button type="button" class="button8"> Ruby</button></ul>
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+                <div class="listproject">
+                    <h2 class="art">Profesional Art Printing Data</h2>
+                    <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button type="button" class="button6"> html</button></ul>
+                        <ul><button type="button" class="button7"> bootstrap</button></ul>
+                        <ul><button type="button" class="button8"> Ruby</button></ul>
+                    </li>
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+            <div class="listproject">
+                <h2 class="art">Profesional Art Printing Data</h2>
+                <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                    required. has been the
+                    industry's standard</p>
+                <li class="listbutton2">
+                    <ul><button type="button" class="button6"> html</button></ul>
+                    <ul><button type="button" class="button7"> bootstrap</button></ul>
+                    <ul><button type="button" class="button8"> Ruby</button></ul>
+                </li>
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
```
Comment (christianbiring1) [inline index.html]: - [ ] The button tag inside the list doesn't have any semantic meaning here. Kindly remove that.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+                            <ul class="item2">
+                                <p>Ruby</p>
+                            </ul>
+                            <ul class="item3">
+                                <p>Html</p>
+                            </ul>
+                            <ul class="item4">
+                                <p>Css</p>
+                            </ul>
+                        </li>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use semantic tag following the Web3C standard.
`It's supposed to be:`
`<ul>`
    `<li>JavaScript</li>`
     `<li>Ruby</li>`
`</ul>`
`And not the other way around. Please consider changing that.`

File: index.html (hunk 6 above)
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use semantic tag following the Web3C standard.
`It's supposed to be:`
`<ul>`
    `<li>JavaScript</li>`
     `<li>Ruby</li>`
`</ul>`
`And not the other way around. Please consider changing that.`

File: style.css [hunk 10]
```diff
@@ -0,0 +1,424 @@
...
+@media screen and (max-width: 768px) {
+  .header {
+    color: #fff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
```
Comment (christianbiring1) [inline style.css:15]: - [ ] Kindly make sure your page doesn't overlap. The width is supposed to fits in the screen size.
Your design :point_down: 
image
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @stevenmukama , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @stevenmukama , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**